.gitignore

```diff
@@ -145,3 +145,11 @@ match_results.tab
 results.tab
 test.tab
 
+$ VS Code 
```

Comment:
`.gitignore` comments start with `#` not `$`. I'm assuming `$ VS Code` and the other lines starting with `$` are comments? Also let's not exclude `ipynb` files. If you have some test code in a notebook and don't want to add it to the repo, don't stage it for commit please.